packages/@o3r/design/src/core/design-token/renderers/design-token.renderer.interface.ts

```diff
@@ -17,10 +17,20 @@ export type DesignContentFileUpdater = (variables: string[], file: string, style
  */
 export type TokenDefinitionRenderer = (tokenStructure: DesignTokenVariableStructure, variableSet: Map<string, DesignTokenVariableStructure>) => string | undefined;
 
+/**
+ * Options of the Design Token list transform function
+ */
+export interface DesignTTokenListTransformOptions {
+  /**
+   * Renderer the name of generated variable (without the prefix required by the target language)
+   */
+  tokenVariableNameRenderer?: TokenKeyRenderer;
+}
+
 /**
  * Function defining the way the variable should be sorted before being generated
  */
-export type TokenListTransform = (variableSet: DesignTokenVariableSet) => (tokens: DesignTokenVariableStructure[]) => DesignTokenVariableStructure[];
+export type DesignTokenListTransform = (variableSet: DesignTokenVariableSet, options?: DesignTTokenListTransformOptions) => (tokens: DesignTokenVariableStructure[]) => DesignTokenVariableStructure[];
```

Comment:
Isn't it a breaking change ?

Reply:
Nop, the original `TokenListTransform` is still on **mian** branch